target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -177,7 +179,9 @@ case "$FIRMWARE" in
 		ath10kcal_extract "ART" 20480 12064
 		ath10kcal_patch_mac_crc $(mtd_get_mac_binary dnidata 12)
 		;;
+	engenius,eap1300 |\
```

Comment:
This entry needs to be removed now that it is between the compex,wpj428 and the openmesh,a42.